.github/ISSUE_TEMPLATE/airflow_bug_report.yml

```diff
@@ -25,7 +25,8 @@ body:
         discuss Airflow 1.10, open a [discussion](https://github.com/apache/airflow/discussions) instead!
       multiple: false
       options:
-        - "2.3.1 (latest released)"
+        - "2.3.2 (latest released)"
+        - "2.3.1"
```

Comment:
BTW. The list is getting longer and longer. I think we shoudl trim it down as it gets very difficult to reach "main" : )

Reply:
Maybe we should use 2.0.* and 2.1.* instead of full list for those versions?

Reply:
That works for me.

Reply:
Works for me too. Should we do it here or in a separate PR?

Reply:
Separate PR likely better :)

Reply:
We can also move main to the top.

Reply:
It's kinda good "main" is at the end (this template is mostly for users and "main" is really something for contributors)